pkg/controller/namespace/deletion/status_condition_utils.go

```diff
@@ -48,16 +48,22 @@ var (
 		v1.NamespaceDeletionDiscoveryFailure,
 		v1.NamespaceDeletionGVParsingFailure,
 		v1.NamespaceDeletionContentFailure,
+		v1.NamespaceContentRemaining,
+		v1.NamespaceFinalizersRemaining,
 	}
 	okMessages = map[v1.NamespaceConditionType]string{
 		v1.NamespaceDeletionDiscoveryFailure: "All resources successfully discovered",
 		v1.NamespaceDeletionGVParsingFailure: "All legacy kube types successfully parsed",
-		v1.NamespaceDeletionContentFailure:   "All content successfully deleted",
+		v1.NamespaceDeletionContentFailure:   "All content successfully deleted, may be waiting on finalization",
+		v1.NamespaceContentRemaining:         "All content successfully removed",
+		v1.NamespaceFinalizersRemaining:      "All content successfully removed",
```

Comment:
s/All content/All finalizer/?

Reply:
> s/All content/All finalizer/?

This is a condition about content in the namespace (like configmaps for instance), that have finalizers on them. We've seen situations in the wild where product B puts a finalizer on configmap/foo and forgets to remove it.

Reply:
Do you still prefer "All finalizers"?  I have no real preference, just want to make sure the intent is clear.

Reply:
i think it's ok to use 'All content', but probably still better to use slightly different wording for the two messages to tell them apart. wdyt?

Reply:
Ideas:
* `No content-blocking finalizers remain`
* `All content-preserving finalizers finished`